### PR TITLE
Minor review changes for revision history 

### PR DESCRIPTION
**Pull request by hrishin**
Accommodate minor review changes from https://github.com/fabric8io/fabric8-maven-plugin/pull/1025/files/ee2dbd348ce2c364f394c8c188c4773eb4a5efab#diff-6cdf58b53e8707096bf08495bf352200R55

### CODE REVIEW

File: enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/RevisionHistoryEnricher.java [hunk 1]
```diff
@@ -52,15 +48,14 @@ public RevisionHistoryEnricher(EnricherContext buildContext) {
...
+        final Integer maxRevisionHistories = Configs.asInt(getConfig(Config.limit));
 
         log.info("Adding revision history limit to %s", maxRevisionHistories);
 
         builder.accept(new TypedVisitor<DeploymentBuilder>() {
             @Override
             public void visit(DeploymentBuilder item) {
-                DeploymentFluent.SpecNested<DeploymentBuilder> spec =
-                        item.getSpec() == null ? item.withNewSpec() : item.editSpec();
+                DeploymentFluent.SpecNested<DeploymentBuilder> spec = item.editOrNewSpec();
```
Comment (rhuss) [inline enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/RevisionHistoryEnricher.java]: Actually you could even elegantly nest it like in 

```java
item.editOrNewSpec()
      .withRevisionHistoryLimit(maxRevisionHistories)
    .endSpec();
```

so you don't need to declare the lengthy variable.

Comment (hrishin) [reply enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/RevisionHistoryEnricher.java]: fixed

**Comment by rhuss** [comment]
[merge]

**Comment by hrishin** [comment]
Thanks for reviews :+1: 
My pleasure :)

**Pull request merged**